operator/v1/types_ingress.go

```diff
@@ -188,6 +188,9 @@ const (
 
 	// Private does not publish the ingress controller.
 	PrivateStrategyType EndpointPublishingStrategyType = "Private"
+
+	// NodePort publishes the ingress controller using a Kubernetes NodePort Service.
+	NodePortStrategyType EndpointPublishingStrategyType = "NodePort"
```

Comment:
`NodePortStrategyType`→`NodePortServiceStrategyType` for consistency with `LoadBalancerServiceStrategyType`?

Reply:
Should the name be NodePortService everywhere instead of NodePort?

Reply:
To be consistent, yeah.

Reply:
Updated throughout for consistency with LoadBalancerService.

Reply:
Well, I take that partly back: We use the name `LoadBalancerStrategy` for the union type for the `LoadBalancerService` strategy type, so perhaps we should use the name `NodePortStrategy` for the new union type.

Reply:
Updated to be consistent with LoadBalancer's inconsistencies 😁